### PR TITLE
Make ENS contacts behave like 0x address contacts

### PR DESCRIPTION
**Pull request by nicoglennon**
Contacts added via ENS will display their ENS address just like contacts added via 0x address in the contacts list and in the create/edit modal.

PoW: https://www.loom.com/share/bc28150c1e64417ea85121b370a5d03b

### CODE REVIEW

File: src/components/send/SendHeader.js [hunk 1]
```diff
@@ -83,19 +88,23 @@ export default function SendHeader({
...
+      color = getRandomColor();
     }
 
     navigate(Routes.MODAL_SCREEN, {
       additionalPadding: true,
       address: recipient,
       color,
-      contact: isEmpty(contact.address) ? false : contact,
+      contact: isEmpty(contact.address)
+        ? recipient.match(/^(0x)?[0-9a-fA-F]{40}$/)
```
Comment (brunobar79) [inline src/components/send/SendHeader.js:100]: You can use `import { isValidAddress } from 'ethereumjs-util';` instead

**Review by brunobar79 — APPROVED** [review]
LGTM! 
Would love for @skylarbarrera  to take a look before merging

**Comment by brunobar79** [comment]
Also keep an eye on e2e failures!

**Review by mikedemarais — APPROVED** [review]
noice

**Pull request merged**